src/doodle/doodledOver.js

```diff
@@ -1,6 +1,19 @@
-const IGNORE_PERCENT_THRESHOLD = 0.8; // Ignore elements that cover at least this % of the region
+const APPEARANCE_THRESHOLD = 0.75; // How much of a line needs to appear in that element to consider it
+
+const IGNORE_PERCENT_THRESHOLD = 0.5; // Ignore elements that cover at least this % of the region
```

Comment:
Do you think that this number will be good for most pages?

Reply:
Probably? It's easily changed in the future if it turns out to not be great.